evm/src/fixed_recursive_verifier.rs

```diff
@@ -452,11 +464,50 @@ where
             }
         }
 
+        // Extra products to add to the looked last value
+        let mut extra_looking_products = Vec::new();
+        for _ in 0..NUM_TABLES {
+            extra_looking_products.push(Vec::new());
+        }
+
+        // Arithmetic
+        for _ in 0..stark_config.num_challenges {
```

Comment:
Can this be done using `extra_looking_products.push(vec![builder.constant(F::ONE); stark_config.num_challenges]);`? Or even combining the four loops into one `vec![vec![...]]` expression?

Reply:
It could be, the idea is that specific functions for the other CTLs could be added here. I can't see any use case now or in the future but I wanted to leave the option to.